pkg/installer/installer.go

```diff
@@ -88,7 +88,7 @@ func Install(ctx context.Context, args *Args) (err error) {
 	case args.DryRun:
 		if args.KubeVersion == nil {
 			// default higher version
-			if args.KubeVersion, err = version.ParseSemantic("1.27.0"); err != nil {
+			if args.KubeVersion, err = version.ParseSemantic("1.30.0"); err != nil {
```

Comment:
1. `v1.30.0` is just a week old and we don't want to force to use it. Instead you could stick to `v1.29.0`
2. You could fix https://github.com/minio/directpv/blob/master/cmd/kubectl-directpv/install.go#L53